### PR TITLE
Tunnel support

### PR DESCRIPTION
**Pull request by mihalis68**
Rudimentary tunnel support so you can load horizon from the host after doing a build by simply

```
make tunnel
```

from the root dir after make create all.

### CODE REVIEW

File: virtual/bin/make-tunnel.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+else
+    host_ip=`hostname -i`
+fi
+
+cd $virtual_dir
+
+vagrant ssh-config ${bootstrap_name} > /tmp/ssh-config.$$
+
+sudo ssh -C -F /tmp/ssh-config.$$ \
+     -L ${host_ip}:8443:10.10.254.254:443 \
```
Comment (justinjpacheco) [inline virtual/bin/make-tunnel.sh]: is there any way we can get the vip dynamically? this value will change depending on the environment

Comment (mihalis68) [reply virtual/bin/make-tunnel.sh]: Probably. I'll look into that

Comment (kamidzi) [reply virtual/bin/make-tunnel.sh]: I maintain ports should not be hardcoded and the port forwarding should be dynamic instead; in order to use horizon, one would then need to configure the ssh connection as the socks proxy. This way any dns resolutions should work reliably within horizon.

**Comment by mihalis68** [comment]
Replaced by https://github.com/bloomberg/chef-bcpc/pull/1487 which is better

**Pull request closed without merging**